sklearn/linear_model/tests/test_omp.py

```diff
@@ -150,6 +151,7 @@ def test_orthogonal_mp_gram_readonly():
 
 
 # FIXME: 'normalize' to be removed in 1.4
+@fails_if_unstable_openblas
 @pytest.mark.filterwarnings("ignore:The default of 'normalize'")
 def test_estimator():
```

Comment:
#21361 is about the following test from `test_common.py`. It's not related to `test_omp.py`.

```
test_common.py::test_estimators[NuSVC()-check_classifiers_train(readonly_memmap=True)]
```

Reply:
I do not know what happens in @jjerphan's head.